pubspec.yaml

```diff
@@ -9,43 +9,47 @@ environment:
   sdk: ">=2.7.0 <3.0.0"
 
 dependencies:
-  connectivity_plus: ^1.0.2
-  cupertino_icons: ^1.0.2
-  date_time_picker: ^2.0.0
-  duration_picker: ^1.1.0+1
   flutter:
     sdk: flutter
-  flutter_config: ^2.0.0
-  flutter_countdown_timer: ^4.1.0
-  flutter_local_notifications: ^9.2.0
-  flutter_polyline_points: ^1.0.0
+
+  cupertino_icons: ^1.0.2
   fluttertoast: ^8.0.3
-  geocoder: ^0.2.1
+  connectivity_plus: ^1.0.2
   geolocator: ^7.0.1
-  get_it: ^7.2.0
   google_maps_flutter: ^2.0.6
-  graphql_flutter: ^5.0.0
-  hive: ^2.0.4
   intl: ^0.17.0
-  location: ^4.3.0
+  date_time_picker: ^2.0.0
   modal_progress_hud: ^0.1.3
-  overlay_support: ^1.2.1
-  path_provider: ^2.0.8
-  provider: ^6.0.2
-  rxdart: ^0.26.0
-  share_plus: ^3.0.4
+  hive: ^2.0.4
+  path_provider: ^2.0.2
   shared_preferences: ^2.0.6
-  sizer: ^2.0.15
-  skeleton_text: ^1.0.0
-  sliding_up_panel: ^2.0.0+1
+  graphql_flutter: ^5.0.0
+  get_it: ^7.1.3
+  provider: ^5.0.0
   uni_links: ^0.5.1
-
+  share_plus: ^3.0.4
```

Comment:
Sorted dependencies have been randomized.

Reply:
I have Sorted dependencies in the latest commit.